presentation/src/main/java/com/nexters/knownknowns/presentation/feature/home/dialog/PopUpDialog.kt

```diff
@@ -0,0 +1,131 @@
+) {
+    val pagerState = rememberPagerState(
+        pageCount = { cardItems.size },
+        initialPage = cardIndex
+    )
+
+    val configuration = LocalConfiguration.current
+    val screenWidth = configuration.screenWidthDp.dp
+
+    val pageSize = screenWidth * CARD_WIDTH_RATIO
+    val horizontalPadding = (screenWidth - pageSize) / 2
+
+    Dialog(
+        onDismissRequest = onDismissRequest,
+        properties = DialogProperties(usePlatformDefaultWidth = false)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .noRippleClickable(onClick = onDismissRequest),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(modifier = Modifier.weight(217f))
+            HorizontalPager(
+                state = pagerState,
+                pageSize = PageSize.Fixed(pageSize),
+                pageSpacing = 12.dp,
+                contentPadding = PaddingValues(horizontal = horizontalPadding),
+                key = { cardItems[it].id },
+            ) { pageIndex ->
+                val item = cardItems[pageIndex]
+
+                PopUpItem(
+                    title = item.title,
+                    keyword = item.keyword,
+                    letter = item.letter,
+                    titleColor = KnownKnownsTheme.colors.statePositivePrimary, // TODO: 색상 분기처리하기 by 이유빈
+                    summary = item.summary,
+                    onClick = onDismissRequest,
+                )
+            }
+            Spacer(modifier = Modifier.height(16.dp))
+            Indicator(
+                pageCount = cardItems.size,
+                pageIndex = pagerState.currentPage
+            )
+            Spacer(modifier = Modifier.height(42.dp))
+            Image(
+                painter = painterResource(id = R.drawable.ic_popup_dismiss),
+                contentDescription = "pop up dismiss button",
+                modifier = Modifier
+                    .background(
+                        color = KnownKnownsTheme.colors.backgroundBase.copy(alpha = 0.1f),
+                        shape = CircleShape
+                    )
+                    .padding(13.dp)
+                    .noRippleClickable(onClick = onDismissRequest)
+            )
+            Spacer(modifier = Modifier.weight(124f))
+        }
+    }
+}
+
+@Composable
+private fun Indicator(
+    pageCount: Int,
+    pageIndex: Int,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+    ) {
+        repeat(pageCount) { index ->
+            val isSelected = index == pageIndex
+
+            Box(
+                modifier = Modifier
+                    .padding(4.dp)
+                    .clip(CircleShape)
+                    .size(8.dp)
+                    .background(
+                        if (isSelected) KnownKnownsTheme.colors.backgroundBase
+                        else KnownKnownsTheme.colors.backgroundBase.copy(alpha = 0.3f)
```

Comment:
사소한 건데 나라면 아마 이런 식으로 쓸 거 같아!
```suggestion
                        KnownKnownsTheme.colors.backgroundBase.copy(alpha = if (isSelected) 1f else 0.3f)
```

Reply:
사소한 코드리뷰 환영입니다 !! 좋아요좋아요